tests/test_client/tests.py

```diff
@@ -549,6 +550,21 @@ def test_force_login_without_backend(self):
         self.assertEqual(response.context['user'].username, 'testclient')
         self.assertEqual(self.u1.backend, 'django.contrib.auth.backends.ModelBackend')
 
+    @override_settings(AUTHENTICATION_BACKENDS=[
+        'test_client.auth_backends.BackendWithoutGetUserMethod',
+        'django.contrib.auth.backends.ModelBackend'
+    ])
+    def test_force_login_with_backend_missing_get_user(self):
+        """
+        In force_login() use only authentication backend which has get_user()
+        method.
+        """
+        self.client.force_login(self.u1)
+        request = HttpRequest()
```

Comment:
Unless I'm missing the purpose of the rest of the test, it seems sufficient to replace from here down with `self.assertEqual(self.u1.backend, 'django.contrib.auth.backends.ModelBackend')`.